core/templates/dev/head/components/static_graph_directive.html

```diff
@@ -0,0 +1,95 @@
+<style>
+  .oppia-static-graph .pannable-rect {
+    cursor: move;
+    fill: none;
+    fill-opacity: 0;
+    pointer-events: all;
+  }
+  .oppia-static-graph path.link {
+    fill: none;
+    stroke: rgb(179, 179, 179);
+    stroke-width: 3px;
+  }
+  .oppia-static-graph text {
+    font: 12px sans-serif;
+    pointer-events: none;
+  }
+  .oppia-static-graph text.shadow {
+    stroke: #fff;
+    stroke-width: 3px;
+    stroke-opacity: .8;
+  }
+  .oppia-static-graph .node rect {
+    fill: #fff;
+    pointer-events: all;
+    stroke: rgb(0, 0, 0);
+    -moz-transition: all 0.15s;
+    -webkit-transition: all 0.15s;
+    -o-transition: all 0.15s;
+    transition: all 0.15s;
+  }
+  .oppia-static-graph .node rect.terminal-node {
+    fill: green; /* terminal nodes are green to signify them */
+  }
+  .oppia-static-graph .node rect.bad-node {
+    fill: #F35F55;
+  }
+  .oppia-static-graph .node rect.current-node:hover,
+  .oppia-static-graph .node rect.init-node:hover,
+  .oppia-static-graph .node rect.terminal-node:hover,
+  .oppia-static-graph .node rect.bad-node:hover,
+  .oppia-static-graph .node rect.normal-node:hover {
+    fill: #C8C4C4;
+  }
+  .oppia-static-graph .clickable {
+    cursor: pointer;
+  }
+  .oppia-static-graph .legend-bar {
+    fill: url('#nodegradient');
+    stroke: black;
+    stroke-width: 0.5;
+  }
+
+</style>
+
+<div style="height: <[getGraphHeightInPixels()]>px;" ng-show="graphLoaded">
+
+  <svg class="oppia-static-graph" width="100%" height="100%">
+        <g>
+          <g>
+            <g ng-repeat="link in augmentedLinks">
+              <path class="link protractor-test-link" ng-if="link.d" ng-attr-d="<[link.d]>" marker-end="url(#arrowhead)" ng-attr-style="<[link.style]>">
+              </path>
+            </g>
+
+            <g class="node protractor-test-node" ng-repeat="node in nodeList">
+              <rect rx="2" ry="2" ng-attr-height="<[node.height]>" ng-attr-width="<[node.width]>"
+                    class="protractor-test-node-background <[node.nodeClass]>"
+                    ng-attr-x="<[node.x0]>" ng-attr-y="<[node.y0]>"
+                    ng-class="{'clickable': canNavigateToNode(node.id)}"
+                    ng-attr-style="<[node.style]>"
+                    ng-click="onClickFunction(node.id)">
+              </rect>
+              <title><[getNodeTitle(node)]></title>
+              <text text-anchor="middle" ng-attr-x="<[node.xLabel]>" ng-attr-y="<[node.yLabel]>" class="protractor-test-node-label">
+                <tspan alignment-baseline="central"><[node.label]></tspan>
```

Comment:
hmm this is kind of puzzling -- injecting $filter should have done the job. I'm not sure why this didn't work, you might have to poke around further.